src/rollup/po2i18n.js

```diff
@@ -0,0 +1,24 @@
+const createFilter = require('@rollup/pluginutils').createFilter;
+const parsePoFile = require('../parsePoFile/parsePoFile');
+
+module.exports = function po2I18next ({
```

Comment:
This is great! 
Can you document its usage on the README so projects can implement it in their build

Reply:
Yes, updated README in 823ea819a1f39e795dd003acd9ec567ec9699e6d